doc(release): Prepare release notes for collect 22.10 (#2637)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md

## Centreon Collect

### 22.10.5

> Release date: `September 25, 2023`

Reviewer: ```suggestion
Release date: `September 26, 2023`
```

#### Centreon Broker

##### Bug fixes

> - Fixed a bug that caused RRD graphs not to display properly when the check interval was longer than 15 minutes.

Reviewer: ```suggestion
- Fixed a bug that caused RRD graphs to not display properly when the check interval was longer than 15 minutes.
```

> - Index data INSERT request have been replaced by INSERT ON DUPLICATE KEY UPDATE and check_interval is always filled.

Reviewer: ```suggestion
- Index data INSERT requests have been replaced by INSERT ON DUPLICATE KEY UPDATE and check_interval is always filled in.
```

> - Index data INSERT requests have been replaced by INSERT ON DUPLICATE KEY UPDATE and check_interval is always filled in.

Reviewer 2: ```suggestion
- Fixed a bug that caused the RRD graphs not to display properly when the check interval was longer than 15 minutes.
```

> -	Fixed an issue that could cause a crash when many pollers were sending check results to a broker.

Reviewer: ```suggestion
-	Fixed an issue that could cause a crash when many pollers were sending check results to Broker.
```

> -	Fixed an issue that could cause a crash when many pollers were sending check results to Broker.

Reviewer: ```suggestion
- Fixed an issue that could cause a crash when many pollers were sending check results to Broker.
```
